### PR TITLE
[CCAP-543] Handle holidays, and holidays pushing expirations to weekends, for 3 business day logic

### PR DESCRIPTION
**Pull request by cram-cfa**
#### 🔗 Jira ticket
https://codeforamerica.atlassian.net/browse/CCAP-543

#### ✅ Completion tasks

- [ ] Added relevant tests
- [ ] Meets acceptance criteria

### CODE REVIEW

File: src/main/java/org/ilgcc/app/utils/ProviderSubmissionUtilities.java [hunk 1]
```diff
@@ -130,12 +150,31 @@ private static String hoursRequested(Map<String, Object> child) {
...
+        ZonedDateTime submittedAt = submittedAtDate.atZoneSameInstant(chicagoTimeZone);
+
+        Integer daysToOffset = DAY_OF_WEEK_WITH_BUSINESS_DAYS_OFFSET.get(submittedAt.getDayOfWeek().toString());
+        ZonedDateTime expiresAt = submittedAt.plusDays(daysToOffset);
+
+        LocalDate submittedAtLocalDate = submittedAt.toLocalDate();
+        LocalDate expiresAtLocalDate = expiresAt.toLocalDate();
+
+        for (var holiday : HOLIDAYS) {
+            if ((holiday.isAfter(submittedAtLocalDate) && holiday.isBefore(expiresAtLocalDate)) || holiday.isEqual(
```
Comment (analoo) [inline src/main/java/org/ilgcc/app/utils/ProviderSubmissionUtilities.java:162]: At first I was wondering why we aren't considering if the date submitted is equal to a holiday but then I realize that we start counting day 1 the day after so this logic makes sense to me.
So, if there is a holiday in the next 3 days, we are giving folks an extra day.

File: src/main/java/org/ilgcc/app/utils/ProviderSubmissionUtilities.java [hunk 2]
```diff
@@ -130,12 +150,31 @@ private static String hoursRequested(Map<String, Object> child) {
...
+                    expiresAtLocalDate)) {
+                expiresAt = expiresAt.plusDays(1);
+            }
+        }
+
+        while (WEEKENDS.contains(expiresAt.getDayOfWeek())) {
```
Comment (analoo) [inline src/main/java/org/ilgcc/app/utils/ProviderSubmissionUtilities.java:168]: then, if our new date puts the day in a weekend, we keep adding days until we are not in a weekday.

So if a person submits on Tuesday, they should have until Friday.

If Friday is a holiday, then saturday would be the due date, but saturday is not a business day so we go to sunday. then to monday.

**Review by analoo — APPROVED** [review]
Really appreciate the tests

File: src/main/java/org/ilgcc/app/utils/ProviderSubmissionUtilities.java (hunk 2 above)
Comment (cram-cfa) [reply src/main/java/org/ilgcc/app/utils/ProviderSubmissionUtilities.java:168]: I am going to add a couple comments in the code to clarify that this is what is happening!

**Pull request merged**